### PR TITLE
Reference env variables if present for dev auth config settings

### PR DESCRIPTION
**Pull request by greg-murray-volusion**
* **Please check if the PR fulfills these requirements**
    + [ ] Tests for the changes have been added (for bug fixes / features)
    + [ ] Docs have been added / updated (for bug fixes / features)
    + [ ] Addresses [Github issue #n](https://github.com/volusion/element-cli/issues/n) (please replace `n` with the corresponding issue number -- no issue for this PR? You can create one [here](https://github.com/volusion/element-cli/issues/new) right quick! 🙏 )

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
 
The ability to use environment variables for dev auth connection info.

* **What is the current behavior?**

Currently uses static values in the config file.

* **What is the new behavior (if this is a feature change)?**

Can override these values if needed.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No

* **Other information**:

### CODE REVIEW

File: config.ts [hunk 1]
```diff
@@ -15,9 +15,15 @@ interface AuthConstants {
...
-    loginUrl: "https://materialsandbox.auth0.com/oauth/token",
+    auth0Audience:
+        process.env.ELEMENT_DEV_AUTH0_AUDIENCE ||
+        "https://sandbox.api.material.com",
+    auth0ClientId:
+        process.env.ELEMENT_DEV_AUTH0_CLIENT_ID ||
+        "ffh6DBHImNf7ZoTlL0sRmQU3EQ2iDuUx",
+    loginUrl:
+        process.env.ELEMENT_DEV_LOGIN_URL ||
+        "https://materialsandbox.auth0.com/oauth/token",
```
Comment (santios-volusion) [inline config.ts]: I wonder if we should just keep the production configurations in the repo and control dev vs sandbox with the env variables.

Comment (greg-murray-volusion) [reply config.ts]: Yes, if we are saying 3rd party partners will be using production, then I can remove the sandbox settings and just have prod but with the env variable overrides.

Comment (santios-volusion) [reply config.ts]: Yeah let's do it.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@volusion/element-cli",
-  "version": "3.0.3",
+  "version": "3.0.4",
```
Comment (santios-volusion) [inline package.json]: I know is not ideal, but we have to update the index.ts file to reflect the new version. Or if you want, you can leave it as 3.0.3, we can merge and release with another PR for the CLI coming soon.

Comment (greg-murray-volusion) [reply package.json]: pushed up a commit, thanks!

File: src/utils/network.ts [hunk 3]
```diff
@@ -251,7 +251,13 @@ export const loginRequest = (
     };
 
     if (isVerbose) {
-        logInfo(`\nRequesting ${config.loginUrl}...\n\n`);
+        logInfo(`\nRequesting ${config.loginUrl}...`);
+        logInfo(
+            `with data: ${JSON.stringify(data).replace(
```
Comment (santios-volusion) [inline src/utils/network.ts]: Or do `delete data.password` before calling JSON.stringify.

Comment (greg-murray-volusion) [reply src/utils/network.ts]: I removed the regex, thanks.

**Pull request merged**